ImperatorToCK3/Source/CK3/Titles/LandedTitles.cpp

```diff
@@ -27,14 +27,19 @@ void CK3::LandedTitles::registerKeys()
 		// Pull the titles beneath this one and add them to the lot, overwriting existing ones.
 		auto newTitle = LandedTitles();
 		newTitle.loadTitles(theStream);
+
+		if (titleName.find("b_")==0 && capitalBarony.empty()) // title is a barony, and no other barony has been found in this scope yet
+		{
+			capitalBarony = titleName;
+		}
 		for (const auto& locatedTitle: newTitle.getFoundTitles())
```

Comment:
use braced initialization instead of locatedTitle

Reply:
Did you mean structured binding?

Reply:
Yes, sorry.

Reply:
In that case it's done ✔️